examples/capi-quick-start/nutanix-cluster-cilium-helm-addon.yaml

```diff
@@ -104,9 +104,9 @@ spec:
               cluster:
                 name: ${NUTANIX_PRISM_ELEMENT_CLUSTER_NAME}
                 type: name
-              image:
-                name: ${NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME}
-                type: name
+              imageLookup:
```

Comment:
Can `clusterctl` templating support if/else statements to still allow setting `NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME` if set?

Reply:
i don't believe it can.

Reply:
No... it's just simple env var replacement really.